console/src/main/webapp/manager/app/app.es6

```diff
@@ -152,7 +152,7 @@ angular.module('manager', [
     require: 'ngModel',
     restrict: 'A',
     link: (scope, elm, attrs, ctrl) => {
-      const alphanum = v => v && v.match(/^[a-zA-Z-_]+$/)
+      const alphanum = v => v && v.match(/^[A-Z0-9-_]+$/)
       let validator = alphanum
       if (scope.$eval(attrs.uppercased)) {
         validator = v => alphanum(v) && v.toUpperCase() === v
```

Comment:
@tonio does it still needed ?

Reply:
Well the directive can be configured with `uppercased` attribute to check if case should be forced or not.
You may have to check where is this directive used…